src/styles/sharedComponents/buttons/buttonVariants.js

```diff
@@ -0,0 +1,61 @@
+// @flow strict-local
+
+import { StyleSheet } from "react-native";
+import type { TextStyleProp, ViewStyleProp } from "react-native/Libraries/StyleSheet/StyleSheet";
+
+import colors from "../../colors";
+
+const PRIMARY = "#5D8017";
+const PRIMARY_DISABLED = "#C6DC98";
+const WARNING = "#9B1111";
+const WARNING_DISABLED = "#B95F5E";
+const NEUTRAL = "#979797";
+const NEUTRAL_DISABLED = "#D3D3D3";
+
+const viewStyles: { [string]: ViewStyleProp } = StyleSheet.create( {
+  containerDefault: {
+    alignItems: "center",
+    borderRadius: 40,
+    marginVertical: 15,
+    marginHorizontal: 10
```

Comment:
Couple lingering issues here:

1. Using `alignItems: "center"` with react-native-paper's <Button> makes it so only the content of the button is tappable, not the entire background. I think it's safe to just remove that here.
2. I would remove the margins to maximize reusability. If a particular component needs a button with margins, it should be able to add them with a style prop... which means adding a style prop to our `<Button>`.